### ISSUE TITLE
[no-multiple-actions-in-effects] not reporting for *-MapTo operators

### ISSUE THREAD

**Issue by rafaelss95** [timdeschryver/eslint-plugin-ngrx#61]
Example:

```ts
export class Effects {
  readonly test$ = createEffect(() =>
    this.actions$.pipe(ofType(Actions.test), switchMapTo([foo(), bar()])),
  );
}
```

It seems that `*MapTo` operators are missing from:

https://github.com/timdeschryver/eslint-plugin-ngrx/blob/48afe768e45da866681d59df0fe8229aeb7ddf5e/src/utils/selectors/index.ts#L31

**Comment by timdeschryver** [comment]
Good catch @rafaelss95, I haven't thought of these !
Feel free to create a PR if you want, otherwise I'll probably add this change somewhere during next week.
